lib/rest/ipMessaging/v2/service/user.js

```diff
@@ -68,6 +68,8 @@ UserList = function UserList(version, serviceSid) {
    * @param {string} [opts.attributes] -
    *          A valid JSON string that contains application-specific data
    * @param {string} [opts.friendlyName] - A string to describe the new resource
+   * @param {user.webhook_enabled_type} [opts.xTwilioWebhookEnabled] -
+   *          The x_twilio_webhook_enabled
```

Comment:
Need to drop the warning here: https://www.twilio.com/docs/chat/webhook-events#triggering-webhooks-for-rest-api-events

Reply:
Do we own those docs?

Reply:
No, but we can submit the change for review.

Reply:
Added JIRA ticket to do this once headers are supported across all helper libs + cli